pilot/pkg/model/context.go

```diff
@@ -434,7 +434,7 @@ func DefaultMeshConfig() meshconfig.MeshConfig {
 		DisablePolicyChecks:               true,
 		PolicyCheckFailOpen:               false,
 		SidecarToTelemetrySessionAffinity: false,
-		RootNamespace:                     IstioSystemNamespace,
+		RootNamespace:                     ConfigRootNamespace,
```

Comment:
If we are setting this, then you have to move a whole lot of configs like default destination rule to this namespace, make sure it gets created as part of istio install. 

The glitch: helm won’t create a new namespace. It will complain. And it won’t apply a template in some hardcoded namespace either. I tried. 
@sdake knows more.